package.json

```diff
@@ -56,7 +54,28 @@
     "system_uri": {
```

Comment:
As a cosmetic comment, perhaps we can put all the elements that are meant for the dependency downloader under a single key, e.g.
```
"deps_downloader": {
     "system_uri": { ... },
     "safe_app": { ... },
     "ENV": { ... }
}